components/tools/OmeroPy/test/integration/gatewaytest/test_annotation.py

```diff
@@ -357,7 +359,7 @@ def testFileAnnotationSpeed(author_testimg_generated, gatewaywrapper):
     """ Tests speed of loading file annotations. See PR: 4176 """
     try:
         f = NamedTemporaryFile()
-        f.write("testFileAnnotationSpeed text")
+        f.write(("testFileAnnotationSpeed text").encode("utf-8"))
```

Comment:
Would prefixing this with `b` have matched other changes?

Reply:
I personally prefer this approach. I found it easier to notice when scanning the code.
Happy to go with the general view